packages/core/src/components/time-picker/time-picker.scss

```diff
@@ -23,6 +23,10 @@
     height: $large-control-height;
   }
 
+  .flex {
+    display: flex;
+  }
+
```

Comment:
Not using in any render function? Looks like dead css styling. Can you explain this css class?

Reply:
> Not using in any render function? Looks like dead css styling. Can you explain this css class?

resolved